lib/resources/aws/aws_s3_bucket.rb

```diff
@@ -22,6 +22,12 @@ def bucket_acl
     end
   end
 
+  def tags
+    catch_aws_errors do
+      @tags ||= BackendFactory.create(inspec_runner).get_bucket_tagging(bucket: bucket_name).tag_set.map { |tag| { key: tag.key, value: tag.value } }
```

Comment:
Is there any change this will paginate?

Reply:
@clintoncwolfe The AWS API documentation doesn't make any mention of pagination.  
Just to be sure, I assigned tags up to the 50-tag global limit on a test bucket, ran a quick failing tag test, and got output of all 50 key-value pairs.